src/app/screens/LNURLPay/index.tsx

```diff
@@ -101,6 +102,7 @@ function LNURLPay() {
 
     try {
       setLoadingConfirm(true);
+      setDisabled(true);
```

Comment:
Instead of adding a new `disabled`-state-prop could we i.e. do `disabled={loadingConfirm}`?

Reply:
Yeah, that will work.